src/algorithms/ED.jl

```diff
@@ -31,47 +31,51 @@ equivalent to dense eigenvectors.
 
 """
 function exact_diagonalization(H::FiniteMPOHamiltonian;
```

Comment:
We should really add a test for this if we export it. Not really the point of this PR, but we should probably add it to the v0.13 checklist.